src/main/java/seedu/address/model/util/DateTime.java

```diff
@@ -11,7 +11,6 @@
 import java.util.Date;
 
 public class DateTime implements Comparable<DateTime> {
-    public static final String TIME_REGEX = "(([0-1]\\d)|(2[0-3])):([0-5]\\d)";
     public static final String DATE_REGEX = "(([0-2]\\d)|(3[0-1]))-((0[1-9])|(1[0-2]))-(\\d{4})";
     public static final String EARLIEST_DATE_STRING = "01-01-2019 00:00";
     public static final String MESSAGE_CONSTRAINTS =
```

Comment:
Perhaps we should shift this message to the deadline and from class and state explicitly that deadline/from should be in the format ...